test/Datadog.Trace.DuckTyping.Tests/Methods/MethodTests.cs

```diff
@@ -341,6 +341,27 @@ public void DefaultGenericsMethods(object obscureObject)
             Tuple<object, string> wrapper3 = duckAbstract.Wrap<object, string>(null, "World");
             Assert.Null(wrapper3.Item1);
             Assert.Equal("World", wrapper3.Item2);
+
+            duckInterface.ForEachScope<int>(
+                (obj, x) =>
+                {
+                    Assert.Equal(42, x);
+                },
+                42);
```

Comment:
Is it worth adding a few more type tests in here, e.g. `object`, `string` etc?

Reply:
sure, will do!

Reply:
done in ca46c04